src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -114,6 +109,11 @@ public void Start()
 
             logger.Info($"Stopping notifier for '{fullPath}'");
 
+            foreach (var option in options)
+            {
+                option.ExceptionReceived -= OptionsOnExceptionReceived;
```

Comment:
Removal of the handlers we discussed @tmasternak .That's ok as this component adds them on its own.